packages/components/src/internal/components/assay/AssayPicker.tsx

```diff
@@ -227,9 +230,11 @@ export const AssayPicker: FC<AssayPickerProps> = memo(props => {
                                     values={providers}
                                     selected={assaySelectionModel.provider}
                                     onChange={onSelectedProviderChange}
+                                    hasPremium={hasPremium}
+                                    warning={providers?.length > 1 ? undefined : noSpecialtyAssaysWarning}
```

Comment:
It looks like this is the only usage of the `noSpecialtyAssaysWarning` const and since the `providers` are already passed in as a prop to this `SpecialtyAssayPanel` is there a reason why the warning has to be passed in instead of just determined within that `SpecialtyAssayPanel` component? Maybe there is another usage of that comp that wants a different message?

Reply:
We initially implemented this warning feature for arbitrary warnings we might want in the assay picker, but then never used it.  I removed the warning prop and now just showing the warning for this use case in SpecialtyAssayPanel.